crates/typst/src/layout/align.rs

```diff
@@ -316,13 +317,12 @@ cast! {
     HAlignment,
     self => Alignment::H(self).into_value(),
     align: Alignment => match align {
-        Alignment::H(v) => v,
+        Alignment::H(y) => y,
         v => bail!("expected `start`, `left`, `center`, `right`, or `end`, found {}", v.repr()),
     }
 }
 
-/// A horizontal alignment which only allows `left`/`right` and `start`/`end`,
-/// thus excluding `center`.
+/// A horizontal alignment which only allows either ends, thus excluding `center`.
```

Comment:
Same comment here regarding the docs changes.

Reply:
Same response here, but no strong opinion. Up to you.